src/shared/components/app-definitions.ts

```diff
@@ -67,6 +67,37 @@ export const API_LIBRARIES: ApiLibrary[] = [
   },
 ];
 
+export const MODERATION_TOOLS: ToolDetails[] = [
+  {
+    name: "lemmy-bot",
+    link: "https://github.com/SleeplessOne1917/lemmy-bot",
+    description: "A bot library for Lemmy, the fediverse link aggregator.",
+  },
+  {
+    name: "lemmy-bouncer",
+    link: "https://github.com/SleeplessOne1917/lemmy-bouncer",
+    description:
+      "Lemmy automod bot to prevent unvetted users from posting or commenting in a community.",
+  },
```

Comment:
I would move this to the bottom as it as very limited functionality and use case compared to others.